build.go

```diff
@@ -218,10 +218,17 @@ func main() {
 		os.Setenv("GOPATH", gopath)
 		log.Println("GOPATH is", gopath)
 	} else {
-		gopath, _ = filepath.EvalSymlinks(gopath)
+		inside := false
 		wd, _ := os.Getwd()
 		wd, _ = filepath.EvalSymlinks(wd)
-		if filepath.Join(gopath, "src/github.com/syncthing/syncthing") != wd {
+		for _, p := range filepath.SplitList(gopath) {
+			p, _ = filepath.EvalSymlinks(p)
+			if filepath.Join(p, "src/github.com/syncthing/syncthing") == wd {
+				inside = true
+				break
+			}
+		}
+		if !inside {
 			fmt.Println("You are running this outside of GOPATH/src/github.com/syncthing/syncthing, this might cause failure!")
```

Comment:
While you're at it, maybe tweak this to something softer and more grammatical? (Yes, me bikeshedding messages again)

`You seem to have GOPATH set but the Syncthing source not placed correctly within it. Proceed with caution.` or something?

Reply:
Sure thing. Please go ahead and bikeshed my language as much as you can. I am decent at correcting somebody else's messages but rubbish at producing good ones myself.